src/coalgebras/itreeTauScript.sml

```diff
@@ -638,6 +638,12 @@ Proof
   rw[Once itree_unfold,FUN_EQ_THM]
 QED
 
+Theorem itree_bind_left_identity:
```

Comment:
If not one already, this could be made an automatic rewrite.

Reply:
I just removed it since it's the same as itree_bind_thm anyways, which is probably better left manual in case `k x` expands to another bind